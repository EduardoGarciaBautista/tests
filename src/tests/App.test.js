const { default: App } = require('../App');
import '@testing-library/jest-dom';
import React from 'react';
import { shallow } from 'enzyme';



describe('Pruebas App', () => {

    test('should show <App></App>', () => {
        const greet = 'Hola soy Ed';
        const wrapper = shallow(<App greet={greet}/>);

        expect(wrapper).toMatchSnapshot();
    });

    test('should shpw sub by props', () => {
        const greet = 'Hola soy Ed';
        const sub = 'Un subtitulo';
        const wrapper = shallow(
        <App 
        greet={greet}
        sub={sub}
         />);

    
         const text = wrapper.find('p').text();
         console.log(text);

         expect(text).toBe(sub);

    });
    
    
});