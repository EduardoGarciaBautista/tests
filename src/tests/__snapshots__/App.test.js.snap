// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas App should show <App></App> 1`] = `
<Fragment>
  <h1>
    Hola soy Ed
  </h1>
  <p>
    Sub default
  </p>
</Fragment>
`;

exports[`Pruebas App should shpw sub by props 1`] = `
<Fragment>
  <h1>
    Hola soy Ed
  </h1>
  <p>
    Un subtitulo
  </p>
</Fragment>
`;
